src/Examples/Akka.Hosting.Asp.LoggingDemo/Program.cs

```diff
@@ -7,11 +7,15 @@
 using Akka.Hosting.Asp.LoggingDemo;
 using Akka.Remote.Hosting;
 using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
 using LogLevel = Akka.Event.LogLevel;
 
 var builder = WebApplication.CreateBuilder(args);
 builder.Services.AddHealthChecks();
 
+// This is required for WithHealthCheck<T> call
+builder.Services.AddTransient<TestHealth>();
```

Comment:
Ah, I need to remove this since it's no longer needed. Will address that in an update.

Reply:
Fixed